cli/src/test/java/com/devonfw/tools/ide/commandlet/VersionGetCommandletTest.java

```diff
@@ -0,0 +1,54 @@
+package com.devonfw.tools.ide.commandlet;
+
+import com.devonfw.tools.ide.cli.CliException;
+import com.devonfw.tools.ide.context.AbstractIdeContextTest;
+import com.devonfw.tools.ide.context.IdeContext;
+import com.devonfw.tools.ide.log.IdeLogLevel;
+import org.junit.jupiter.api.Test;
+
+
+/**
+   * Integration test of {@link VersionGetCommandlet}.
+   */
+public class VersionGetCommandletTest extends AbstractIdeContextTest{
+
+    /**
+     * Test of {@link VersionGetCommandlet} run, when Installed Version is null.
+     */
+    @Test
+    public void testVersionGetCommandletRunThrowsCliExeption(){
+      // arrange
+      String path = "workspaces/foo-test/my-git-repo";
+      IdeContext context = newContext("basic", path, false);
+      VersionGetCommandlet versionGet = context.getCommandletManager().getCommandlet(VersionGetCommandlet.class);
+      versionGet.tool.setValueAsString("java");
+      // act
+      try {
+        versionGet.run();
+      } catch (CliException e) {
+        //assert
+        assertThat(e).isInstanceOf(Exception.class)
+            .hasMessageContaining("Tool java is not installed!");
+        return;
+      }
+      //just a false case if versionGet.run() does not throw the Exception
+      assertThat(true).isFalse();
```

Comment:
The pattern for this is to call `failBecauseExceptionWasNotThrown(CliException.class)` at the end of the `try { ... }` block.

Reply:
implemented